deploy/playbooks/roles/common/tasks/main.yml

```diff
@@ -5,6 +5,16 @@
   file: path={{ app_home }} owner={{ ansible_ssh_user }} group={{ ansible_ssh_user }} state=directory recurse=yes
   register: app_home_created
 
+- name: "create /var/lib database location for {{ app_name }}"
+  sudo: yes
```

Comment:
the `/var/lib/ceph-installer` directory will be owned by `ansible_ssh_user` here.